scripts/zones/Yuhtunga_Jungle/IDs.lua

```diff
@@ -43,6 +43,7 @@ zones[tpz.zone.YUHTUNGA_JUNGLE] =
         NO_COMBINATION              = 7868,  -- You were unable to enter a combination.
         REGIME_REGISTERED           = 10046, -- New training regime registered!
         COMMON_SENSE_SURVIVAL       = 12040, -- It appears that you have arrived at a new survival guide provided by the Adventurers' Mutual Aid Network. Common sense dictates that you should now be able to teleport here from similar tomes throughout the world.
+        SWARM_APPEARED              = 40588, -- A swarm has appeared!
```

Comment:
This ID looks like it should be `7820`. Think everything else looks good though.

Reply:
Small test: 
![image](https://user-images.githubusercontent.com/774909/81509332-12a54b00-930a-11ea-8345-8e99a16c3da8.png)

I guess the higher bit is used for something that hides the NPC who's saying the message?
(`40588 - 2^15 = 7820`)

Reply:
In the capture, it doesn't have the `???: ` infront.

Reply:
Don't pass `true' in the call to `messageText`, that's what's showing the NPC name.

Reply:
[The messageText packet when called with `false` actually adds that offset you've found to the passed-in messageID to set that higher bit.](https://github.com/project-topaz/topaz/blob/release/src/map/packets/message_text.cpp#L36-L38)

Reply:
Aha, interesting. Will change ID to the lower one and pass `false` then.

Question, where did you find the 7820 ID?

Reply:
[POLUtils](https://github.com/Windower/POLUtils)